src/flake8/main/options.py

```diff
@@ -218,7 +223,15 @@ def register_default_options(option_manager: OptionManager) -> None:
         metavar="format",
         default="default",
         parse_from_config=True,
-        help="Format errors according to the chosen formatter.",
+        help=(
+            f"Format errors according to "
+            f"the chosen formatter ({', '.join(sorted(formatter_names))}) "
+            f"or a format string containing %%-style "
+            f"mapping keys ({', '.join(sorted(MAPPING_KEYS))}). "
+            f"For example, "
+            f"``--format=pylint`` or ``--format='%%(path)s %%(code)s'``. "
+            f"(Default: {DEFAULT_REPORTER})."
```

Comment:
%(default)s

Reply:
Right! Addressed in 27ae1ba318f9521275e70681914ea37036850c8e.